ecosystem/typescript/sdk/src/aptos_types/account_address.ts

```diff
@@ -47,6 +47,13 @@ export class AccountAddress {
     return new AccountAddress(res);
   }
 
+  /**
+   * Return a hex string from account Address.
+   */
+  toHexString(): MaybeHexString {
+    return Buffer.from(this.address).toString("hex");
```

Comment:
We cannot use `Buffer` here. Buffer is not available in Browser env. `HexString.fromUint8Array(this.address).hex()`

Reply:
updated